parl/env/compat_wrappers.py

```diff
@@ -15,33 +15,26 @@
 import gym
 import numpy as np
 from parl.utils import logger
+import operator
 
 # BASE_VERSION1 change env.seed() and env.reset()
 # BASE_VERSION2 change env.step()
+# BASE_VERSION3 change env.np_random from .randint() to .integers()
 BASE_VERSION1 = '0.26.0'
 BASE_VERSION2 = '0.25.0'
 BASE_VERSION3 = '0.21.0'
```

Comment:
Can we use some meaningful variable name? such as `V_RESET_CHANGED`, `V_STEP_CHANGED`, `V_NPRANDOM_CHANGED`. It will be more readable.